.github/README.md

```diff
@@ -54,8 +54,8 @@ AstroNvim is an aesthetic and feature-rich neovim config that is extensible and
 - Optional Requirements:
   - [ripgrep](https://github.com/BurntSushi/ripgrep) - live grep telescope search (`<leader>fw`)
   - [lazygit](https://github.com/jesseduffield/lazygit) - git ui toggle terminal (`<leader>tl` or `<leader>gg`)
-  - [NCDU](https://dev.yorhel.nl/ncdu) - disk usage toggle terminal (`<leader>tu`)
-  - [Htop](https://htop.dev/) - process viewer toggle terminal (`<leader>tt`)
+  - [go DiskUsage()](https://github.com/dundee/gdu) - disk usage toggle terminal (`<leader>tu`)
+  - [gotop](https://github.com/xxxserxxx/gotop) - process viewer toggle terminal (`<leader>tt`)
```

Comment:
So you really do prefer gotop, huh 🤔

Reply:
gotop has been around for a long time and has many many years of development and support. I think for this use case I would like to stick with something a little bit more future proof

Reply:
It's not like `bottom` is unstable or something, and in terms of age, they only have a gap of over a year ( [gotop](https://github.com/xxxserxxx/gotop/commit/40775db60b90cd290a206108cc4d22b236be9ba5) / [bottom](https://github.com/ClementTsang/bottom/commit/0cfcf323a1dba844758e01314b923d0d8665554a) )

Reply:
@mehalter I have tested gotop in Windows, and I am experiencing major slowdowns, making it unusable. I am also experiencing this error: https://github.com/xxxserxxx/gotop/issues/4

Reply:
okie, i'll replace it with bottom @sitiom